lti/configuration.rst

```diff
@@ -18,8 +18,7 @@ in the site before adding any related course content. The sitewide
 configuration of the LTI service contains the key and secret obtained from the
 Tool Provider.
 
-The A+ site administrator accesses the admin interface in the URL path
-`/admin <//localhost:8000/admin>`_.
+The A+ site administrator accesses the admin interface.
```

Comment:
This paragraph could still show the URL path `/admin` since not everyone knows it.

Reply:
I will keep the link, but it will point to the https://plus.cs.aalto.fi/admin/ as the manual is now online